src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/x64/transpose.cpp

```diff
@@ -1,159 +1,44 @@
-        }
-        result << ")_";
-        result << "inputOrder=" << CommonTestUtils::vec2str(inputOrder) << "_";
-        result << "netPRC=" << netPrecision.name() << "_";
-        result << "trgDev=" << targetDevice;
-        result << CPUTestsBase::getTestCaseName(cpuParams);
-        return result.str();
-    }
-protected:
-    void SetUp() override {
-        Precision netPrecision;
-        InputShape inputShapes;
-        std::vector<size_t> inputOrder;
-        CPUSpecificParams cpuParams;
-        std::map<std::string, std::string> additionalConfig;
-        std::tie(inputShapes, inputOrder, netPrecision, targetDevice, additionalConfig, cpuParams) = this->GetParam();
-        configuration.insert(additionalConfig.begin(), additionalConfig.end());
-
-        inType = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
-        outType = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
-
-        std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
-
-        selectedType = makeSelectedTypeStr("unknown", inType);
-
-        init_input_shapes({inputShapes});
-
-        auto params = ngraph::builder::makeDynamicParams(inType, { inputDynamicShapes[0] });
-
-        const auto inputOrderOp = std::make_shared<ov::op::v0::Constant>(ov::element::i64,
-                                                                         ov::Shape({inputOrder.size()}),
-                                                                         inputOrder);
-        const auto transpose = std::make_shared<ov::op::v1::Transpose>(params[0], inputOrderOp);
-        transpose->get_rt_info() = getCPUInfo();
-        const ov::ResultVector results{std::make_shared<ov::op::v0::Result>(transpose)};
-
-        function = std::make_shared<ngraph::Function>(results, params, "TransposeLayerCPUTest");
-        functionRefs = ngraph::clone_function(*function);
-    }
-};
-
-TEST_P(TransposeLayerCPUTest, CompareWithRefs) {
-    run();
-    CheckPluginRelatedResults(compiledModel, "Transpose");
-}
 
+namespace CPULayerTestsDefinitions {
+namespace Transpose {
 namespace {
 std::map<std::string, std::string> additional_config;
 
+const auto cpuParams_ndhwc = CPUSpecificParams {{ndhwc}, {}, {}, {}};
+const auto cpuParams_ncdhw = CPUSpecificParams {{ncdhw}, {}, {}, {}};
+
 const auto cpuParams_nChw16c = CPUSpecificParams {{nChw16c}, {}, {}, {}};
 const auto cpuParams_nCdhw16c = CPUSpecificParams {{nCdhw16c}, {}, {}, {}};
 
 const auto cpuParams_nChw8c = CPUSpecificParams {{nChw8c}, {}, {}, {}};
 const auto cpuParams_nCdhw8c = CPUSpecificParams {{nCdhw8c}, {}, {}, {}};
 
-const auto cpuParams_nhwc = CPUSpecificParams {{nhwc}, {}, {}, {}};
-const auto cpuParams_ndhwc = CPUSpecificParams {{ndhwc}, {}, {}, {}};
-
-const auto cpuParams_nchw = CPUSpecificParams {{nchw}, {}, {}, {}};
-const auto cpuParams_ncdhw = CPUSpecificParams {{ncdhw}, {}, {}, {}};
-
 const std::vector<InferenceEngine::Precision> netPrecisions = {
```

Comment:
We are not testing blocked for I8 and FP32 on x86 after this change.
I guess we should keep all the precisions.

Reply:
[common/transpose.cpp](https://github.com/openvinotoolkit/openvino/pull/17322/files#diff-729cb4b59802f26d60219cf563c5dba507d04933a229143400be0f1ccc3f3c69)
Tests on the lines 37-65 cover I8 and FP32